test/metabase/api/preview_embed_test.clj

```diff
@@ -577,3 +577,62 @@
               (is (= {:values          [["African"] ["American"] ["Asian"]]
                       :has_more_values false}
                      (mt/user-http-request :rasta :get 200 url))))))))))
+
+(deftest parameter-values-are-parsed-and-query-suceeds-test
+  (testing "embedding endpoint parameter values are parsed when sensible. (#27643)"
+    (embed-test/with-embedding-enabled-and-new-secret-key
+      (mt/dataset places-cam-likes
+        (mt/with-temp [:model/Card {card-id :id :as card} {:dataset_query
```

Comment:
`mt/native-query` would save you a couple tokens of typing, but NBD

Reply:
TIL. 
I'll leave this be, but remember that for future tests :)